pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -322,6 +324,21 @@ func (exp *flowExporter) sendDataRecord(dataRec ipfix.IPFIXRecord, record flowex
 			} else {
 				_, err = dataRec.AddInfoElement(ie, "")
 			}
+		case "destinationClusterIP":
+			if record.Conn.DestinationServicePortName != "" {
+				_, err = dataRec.AddInfoElement(ie, record.Conn.TupleOrig.DestinationAddress)
+			} else {
+				// Sending dummy IP as IPFIX collector expects constant length of data for IP field.
+				// We should probably think of better approach as this involves customization of IPFIX collector to ignore
+				// this dummy IP address.
+				_, err = dataRec.AddInfoElement(ie, net.IP{0, 0, 0, 0})
```

Comment:
newbie question: is it not possible to simply omit the information element in this case? Or will the collector complain if some elements are missing?

Reply:
Yes collector checks that if number of elements in a data record matches to originally sent template record or not.